frontend-react/src/components/MessageTracker/MessageDetails.tsx

```diff
@@ -0,0 +1,185 @@
+            itemCount: 2,
+            itemCountBeforeQualityFiltering: 5,
+            filteredReportRows: [
+                "Filtered out item Alaska1",
+                "Filtered out item Alaska2",
+                "Filtered out item Alaska4",
+            ],
+            filteredReportItems: [
+                {
+                    filterType: "QUALITY_FILTER",
+                    filterName: "hasValidDataFor",
+                    filteredTrackingElement: "Alaska1",
+                    filterArgs: ["patient_dob"],
+                    message: "Filtered out item Alaska1",
+                },
+            ],
+            sentReports: [],
+            sending_at: "",
+        },
+        {
+            organization: "Hawaii Public Health Department",
+            organization_id: "hi-phd",
+            service: "elr",
+            itemCount: 2,
+            itemCountBeforeQualityFiltering: 5,
+            filteredReportRows: [
+                "Filtered out item Hawaii6",
+                "Filtered out item Hawaii7",
+                "Filtered out item Hawaii9",
+            ],
+            filteredReportItems: [
+                {
+                    filterType: "QUALITY_FILTER",
+                    filterName: "hasValidDataFor",
+                    filteredTrackingElement: "Hawaii6",
+                    filterArgs: ["specimen_type"],
+                    message: "Filtered out item Hawaii6",
+                },
+                {
+                    filterType: "QUALITY_FILTER",
+                    filterName: "hasValidDataFor",
+                    filteredTrackingElement: "Hawaii7",
+                    filterArgs: ["specimen_type"],
+                    message: "Filtered out item Hawaii7",
+                },
+                {
+                    filterType: "QUALITY_FILTER",
+                    filterName: "hasValidDataFor",
+                    filteredTrackingElement: "Hawaii9",
+                    filterArgs: ["specimen_type"],
+                    message: "Filtered out item Hawaii9",
+                },
+            ],
+            sentReports: [],
+            sending_at: "",
+        },
+    ];
+
+    // const { messageId } = useParams<MessageDetailsProps>();
+    // TODO: call endpoint with messageId to get errors
+    const qualityFilters = MOCK_MESSAGE_QUALITY_FILTERS;
+
+    return <QualityFilters qualityFilters={qualityFilters} />;
+}
+
+export function MessageDetails() {
+    const navigate = useNavigate();
+
+    return (
+        <>
+            <div
+                className="grid-container margin-bottom-10"
+                data-testid="container"
+            >
+                <div className="margin-bottom-5">
+                    <h1>Message Details</h1>
+                    <Button onClick={() => navigate(-1)} type="button">
+                        Back
+                    </Button>
+                </div>
+                {withCatchAndSuspense(<MessageDetailsSender />)}
```

Comment:
Maybe a design question, but I'm curious if we'd want a spinner for each or one spinner until all data loads? Cc: @fmccaf1 

Regardless, I definitely think we'll want those catches on all of them!

Reply:
Good call, lets discuss.

Reply:
I reached out to Florence to get some input for that discussion as well, noting that neither really slow down our fetching process and either one can be its own useful experience (i.e. seeing data faster vs seeing more complete data all at once). I'll paste her reply here once I have it!